atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -385,6 +389,18 @@ private void addSnapshotListener() {
                 .build()));
   }
 
+  private SnapshotChunk decodeSnapshotChunk(final InstallRequest request) {
```

Comment:
Maybe we can have a legacy util class or something where we keep this code and can it remove it more easily. This might be also easier to track then.